tests/bigdata-test.c

```diff
@@ -114,8 +114,12 @@ static int check_bss()
     return 1;
 }
 
-TEST(Aarch64MinGW, BigDataTest)
+TEST(BigDataTest, HashBigText)
```

Comment:
`Aarch64` should not be present in the test name. This change is just a sneak peak how it should be re-factored. I can either revert it, if declined, or move it to a separate PR, if accepted.

Reply:
why Aarch64 cannot be used in the test name?

Reply:
Those tests are architecture-agnostic as they do not use any specifics of the aarch64 target, e.g. assembly. They target certain aarch64 features though.